lang/cs/Org.Apache.REEF.Common/Runtime/Evaluator/HeartBeatManager.cs

```diff
@@ -84,32 +87,20 @@ public HeartBeatManager(EvaluatorSettings settings, IRemoteIdentifier remoteId)
             }
         }
 
-        [SuppressMessage("StyleCop.CSharp.NamingRules", "SA1307:AccessibleFieldsMustBeginWithUpperCaseLetter", Justification = "Intended to be private, exposed now before using future injection")]
-        public EvaluatorRuntime _evaluatorRuntime { get; set; }
+        public EvaluatorRuntime Evaluatorruntime { get; set; }
 
         [SuppressMessage("StyleCop.CSharp.NamingRules", "SA1307:AccessibleFieldsMustBeginWithUpperCaseLetter", Justification = "Intended to be private, exposed now before using future injection")]
-        public ContextManager _contextManager { get; set; }
+        public ContextManager Contextmanager { get; set; }
```

Comment:
Documented. 
It is used in Evalutor project so cannot be internal. We have REEF-1050 to set up friend relationship. I can modify it after merge with that PR.